### ISSUE TITLE
firebase emulator: firestore.rules not relative to project root

### ISSUE THREAD

**Issue by JakeHedman** [firebase/firebase-tools#1255]
### [REQUIRED] Environment info

**firebase-tools:** 6.9.0

**Platform:** Arch Linux

### [REQUIRED] Test case

```sh
mkdir test
cd test
firebase setup:emulators:firestore
firebase emulators:start --only firestore
```

### [REQUIRED] Steps to reproduce
Try to start the firestore emulator when pwd is not the project root

### [REQUIRED] Expected behavior
Emulator should find firestore.rules

### [REQUIRED] Actual behavior
Emulator checks for firestore.rules relative to the current path and crashes.

```
[2019-05-08T05:14:10.765Z] ----------------------------------------------------------------------
[2019-05-08T05:14:10.768Z] Command:       /usr/bin/node /usr/bin/firebase emulators:start --only firestore --debug
[2019-05-08T05:14:10.768Z] CLI Version:   6.9.0
[2019-05-08T05:14:10.768Z] Platform:      linux
[2019-05-08T05:14:10.768Z] Node Version:  v11.15.0
[2019-05-08T05:14:10.769Z] Time:          Wed May 08 2019 07:14:10 GMT+0200 (Central European Summer Time)
[2019-05-08T05:14:10.769Z] ----------------------------------------------------------------------

[2019-05-08T05:14:10.775Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[2019-05-08T05:14:10.775Z] > authorizing via signed-in user
[2019-05-08T05:14:10.777Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/xxx
 
[2019-05-08T05:14:11.741Z] <<< HTTP RESPONSE 200 content-type=application/json; charset=UTF-8, vary=X-Origin, Referer, Origin,Accept-Encoding, date=Wed, 08 May 2019 05:14:11 GMT, server=ESF, cache-control=private, x-xss-protection=0, x-frame-options=SAMEORIGIN, x-content-type-options=nosniff, server-timing=gfet4t7; dur=888, alt-svc=quic=":443"; ma=2592000; v="46,44,43,39", accept-ranges=none, transfer-encoding=chunked
i  Starting emulators: ["firestore"]
[2019-05-08T05:14:11.764Z] Starting emulator firestore with args {"host":"localhost","port":5002,"rules":"firestore/firestore.rules"}
i  firestore: Logging to firestore-debug.log
[2019-05-08T05:14:11.961Z] Exception in thread "main" 
[2019-05-08T05:14:11.961Z] java.io.FileNotFoundException: firestore/firestore.rules (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)

[2019-05-08T05:14:11.961Z] 	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at com.google.common.io.Files$FileByteSource.openStream(Files.java:131)
	at com.google.common.io.Files$FileByteSource.read(Files.java:161)
	at com.google.common.io.ByteSource$AsCharSource.read(ByteSource.java:512)
	at com.google.cloud.datastore.emulator.firestore.CloudFirestore.main(CloudFirestore.java:64)
[2019-05-08T05:14:11.961Z] 


Error: firestore: emulator has exited with code: 1
```

**Comment by ryanpbrewster** [comment]
Can you clarify whether your project does have a `firestore/firestore.rules` file?

**Comment by JakeHedman** [comment]
@ryanpbrewster: It does have a `firestore/firestore.rules` file and the command works if I run it from the project root. It just crashes if I `cd firestore´ first.

**Comment by ryanpbrewster** [comment]
Ah, cool, this may be as simple as ensuring we manually `resolve` paths from the project root. For now I hope it's not a blocker to run this from the base directory.

Thanks for reporting this issue :)

**Comment by JakeHedman** [comment]
@ryanpbrewster: Not a blocker at all! :) Just wanted to create an issue since it can be a bit tricky to find out why it's crashing due to #1254.

**Comment by samtstern** [comment]
This was fixed in #1249 and will be released soon, for now the safest thing is downgrade to `6.8.0`.  Thanks for reporting!

**Comment by samtstern** [comment]
The fix for this issue has been released in `6.9.1`.
